js/babel-preset-noms/index.js

```diff
@@ -39,8 +38,17 @@ const development = {
     preset('react'),
   ],
   plugins: [
+    plugin('syntax-async-functions'),
+    plugin('syntax-flow'),
+    plugin('transform-async-to-generator'),
+    plugin('transform-class-properties'),
     plugin('transform-es2015-modules-commonjs'),
```

Comment:
Was the problem the spread? Because... you could also just make this:
```
plugins: [
  plugin('transform-es2015-modules-commonjs'),
].concat(commonPlugins)
```